README.md

```diff
@@ -1,9 +1,9 @@
 
 <div align="left">
-  <img src="HowsoLogo.svg">
+  <img src="https://howso.com/assets/images/Howso_Readme.svg">
 </div>
 
-The Howso Engine&trade; is a natively and fully explainable ML engine, serving as an alternative to black box AI neural networks. It’s core features give users data exploration and machine learning capabilities through the creation and use of Trainees that help users store, explore, and analyze the relationships in their data. Howso&trade; leverages an instance-based learning approach with strong ties to the [k-nearest neighbors algorithm](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm) and [information theory](https://en.wikipedia.org/wiki/Information_theory) to scale for real world applications.
+The Howso Engine&trade; is a natively and fully explainable ML engine, serving as an alternative to black box AI neural networks. Its core features give users data exploration and machine learning capabilities through the creation and use of Trainees that help users store, explore, and analyze the relationships in their data. Howso&trade; leverages an instance-based learning approach with strong ties to the [k-nearest neighbors algorithm](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm) and [information theory](https://en.wikipedia.org/wiki/Information_theory) to scale for real world applications.
```

Comment:
Maybe the first sentence could be something like

`... and fully explainable ML engine and toolbox, serving as an alternative  to black box AI.`?

Also, should the lines in this be split up via newlines to avoid this several hundred character long line?

Reply:
I just think that `black box AI neural networks` is a bit cumbersome and not great English

Reply:
That sounds like it would be a long description. The end "consumer" of this one line is on the PyPI.org site to be used in lists of packages. Can we keep it short?

Reply:
Oh wait, you're commenting on the README... my bad.

Reply:
Happy to add some newlines

Reply:
makes sense, I agree

Reply:
newlines added

Reply:
Pushing this out to a new PR

Reply:
Please make a new PR with any README content changes.